Float.FileDownloader.Tests/DownloadRequest.Tests.cs

```diff
@@ -72,5 +74,29 @@ public async Task TestServerNotFound()
             var request = new HttpRequestMessage(HttpMethod.Get, "http://www.floatfloatfloat.float/float");
             await Assert.ThrowsAsync<HttpRequestException>(async () => await DownloadRequest.Download(request, TempFilePath()));
         }
+
+        [Fact]
+        public async Task TestEmptyHandler()
```

Comment:
@EBusch This test needs some sort of assert, otherwise it's not really testing anything